pkg/util/buffpool/bufferpool.go

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package buffpool
```

Comment:
Suggest not use shorthand，maybe `pool` is OK, we can put other pool util to this package.

Reply:
No, it was designed only for `bytes.Buffer`, it will discard those over 1MB targets, and it's based on `sync.Pool`, it's not a traditional pool.

Reply:
PS: there are many 3rd bytes pool libs, for example, https://github.com/valyala/bytebufferpool